src/indexer/migrations/20241009154948_create_tables.sql

```diff
@@ -1,11 +1,11 @@
 -- Add migration script here
 CREATE TABLE blocks (
-    hash TEXT PRIMARY KEY,          -- Corresponds to BlockHash 
-    parent_hash TEXT NOT NULL,      -- Parent block hash (BlockHash)
+    hash BYTEA PRIMARY KEY,          -- Corresponds to BlockHash 
```

Comment:
when doing debug, having a TEXT field is very much easier to query than a BYTEA, I highly recommend to keep a string there